examples/manage/pages/application-detail/index.js

```diff
@@ -19,39 +19,38 @@ class ApplicationDetail extends React.Component {
 
   render() {
     return (
-      <div>
-        <Breadcrumb>
-          <Breadcrumb.Link href="#/applications" icon="arrow-left-fill">
-            Back to Applications
-          </Breadcrumb.Link>
-        </Breadcrumb>
-        <ApplicationPageHeader
-          title="API Explorer Application"
-          type={{
-            name: 'Non Interactive',
-            applicationId: this.props.match.params.applicationId
-          }}
-          logo={
-            <Avatar type="resource" size="xlarge" image={ApplicationTypeImages.non_interactive} />
-          }
-          breadcrumb={{
-            content: 'Applications',
-            link: '/#/applications'
-          }}
-        />
-        <Tabs selected={this.state.selectedTab} onSelect={this.handleSelected.bind(this)}>
-          <Tabs.Tab label="Quick Start">Quickstart</Tabs.Tab>
-          <Tabs.Tab label="Settings" selected>
-            <Settings applicationId={this.props.match.params.applicationId} />
-          </Tabs.Tab>
-          <Tabs.Tab label="Connections">
-            <Connections />
-          </Tabs.Tab>
-          <Tabs.Tab label="Advanced">
-            <Advanced applicationId={this.props.match.params.applicationId} />
-          </Tabs.Tab>
-        </Tabs>
-      </div>
+      <PageLayout>
+        <PageLayout.Header>
+          <ApplicationPageHeader
+            title="API Explorer Application"
+            type={{
+              name: 'Non Interactive',
+              applicationId: this.props.match.params.applicationId
+            }}
+            logo={
+              <Avatar type="resource" size="xlarge" image={ApplicationTypeImages.non_interactive} />
+            }
+            breadcrumb={{
```

Comment:
Does this actually render breadcrumbs :thinking:

Reply:
I don't know, it's really strange, but it was there before unrelated to layouts

Reply:
It shows up as a chromatic diff because the above `<Breadcrumb/>` was removed

Reply:
@siddharthkp I bright back the breadcrumb, can you test it out, please?